values.yaml

```diff
@@ -1,13 +1,17 @@
 # For detailed explanation of each of the configuration settings see
 # https://www.pomerium.io/reference/
 
+nameOverride: pomerium
+fullnameOverride: pomerium
+
 # settings that are shared by all services
 config:
   # routes under this wildcard domain are handled by pomerium
   rootDomain: corp.beyondperimeter.com
-  # existingSecret:
-  # existingConfig:
+  existingSecret: ""
```

Comment:
AFAIK, empty strings are equivalent to [being nil in go templates](https://golang.org/pkg/text/template/#hdr-Actions).

Reply:
Yes, that's correct. Any empty string will result in an empty pipeline value which doesn't generate output.